lib/linguist/languages.yml

```diff
@@ -1251,13 +1251,13 @@ EditorConfig:
   tm_scope: source.editorconfig
   language_id: 96139566
 Edje Data Collection:
-  type: data
+  type: markup
```

Comment:
Please change this back to `data`.

Reply:
I was mostly hoping to get this to show up for a repo's language stats, because there are project which are almost 100% .edc files (themes for enlightenment and terminology for example) which show up on GitHub as Shell or Python or even Makefile projects due to having a single shell or Python script/Makefile and .edc not being considered for statistics.

Examples: https://github.com/simotek/Enlightenment-Themes https://github.com/thewaiter/enlightenment-arc-theme https://github.com/sylveon/terminology-themes

Reply:
Said projects can use [an override](https://github.com/github/linguist#detectable) to include `.edc` files in the repository's language breakdown.

Reply:
Ah, wasn't aware of that.